python/lsst/ts/fbs/utils/maintel/make_fieldsurvey_scheduler.py

```diff
@@ -204,9 +206,8 @@ def get_comcam_sv_targets(exclude: typing.List[str] = []) -> dict:
     target_dict : `dict`
         Dictionary of candidate target names and coordinates.
     """
-    infile = get_data_dir() / "field_survey_centers.yaml"
-    with open(infile) as stream:
+    with open(target_file) as stream:
         targets_dict = yaml.safe_load(stream)
 
-    targets_dict = targets_dict["comcam_sv_targets"]
+    targets_dict = targets_dict["sv_targets"]
```

Comment:
I think you should remove this line and just return `targets_dict`  here, otherwise you might unnecessarily filter out new targets that are later added to the target file in ts_config_ocs.

You could move this filter to the scheduler configuration after loading the targets_dict file.

Reply:
A further simplification would be to remove the "sv_targets" category in the yaml file altogether, since no other categories are defined, and it seems unlikely that any would be used at this stage.